.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
@0candy unfortunately, `support/` fails with a huge number of lint problems on the `master` currently... suggestions?